### DISCUSSION TITLE
Livros e Sites Com Conteúdo de Aprendizagem Sobre HTML5

### DISCUSSION THREAD

**Discussion by lifersonskrzek** [frontendbr/forum#2324] [Perguntas e Respostas]
Olá pessoal, espero que estejam bem!

Gostaria de saber quais **livros** e **sites** vocês recomendam para acelerarmos o aprendizado em **HTML5**, tanto em EN como em PT-BR.

Gosto de conteúdos em vídeo, que na realidade são os melhores para a **prática** da linguagem em si. Mas como estou **na fase de transição** da área de Marketing de Conteúdo para Desenvolvimento Web, ainda estou muito 'colado' em conteúdos **escritos**.

Se puderem deixar links de referência, agradeço.

Grande abraço!

**Comment by felquis** [comment]
uma vez li este livro https://livrosdomaujor.com.br/html5/2ed/index.php e foi bem difícil e interessante de ler.

**Reply by lifersonskrzek** [reply]
Olá @felquis, muito obrigado pela indicação. Vou dar um aolhada, sim, e depois retorno aqui para dizer o que achei. Tamo junto!
